.github/workflows/build-verification.yml

```diff
@@ -48,19 +59,24 @@ jobs:
       - name: Set up Gradle
         uses: gradle/gradle-build-action@v2
       - name: Run unit tests
-        run: ./gradlew test -x signPluginMavenPublication -x signAndroidCacheFixPluginPluginMarkerMavenPublication -Porg.gradle.java.installations.auto-download=false
+        run: >-
+            ./gradlew test
+            -x signPluginMavenPublication
+            -x signAndroidCacheFixPluginPluginMarkerMavenPublication
+            -Porg.gradle.java.installations.auto-download=false
+            -Dpts.mode=${{ needs.generate_versions_pts.outputs.pts-mode }}
+            -PisPTSEnabled=${{ github.event.inputs.pts_enabled }}
         env:
           GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_SOLUTIONS_ACCESS_TOKEN }}
-          ORG_GRADLE_PROJECT_isPTSEnabled: ${{ github.ref_name != 'main' }}
```

Comment:
Is it possible to use variable here to set the remaining tests mode inline instead of having an action to set it?

something like `pts.mode: if ${{ github.ref_name == 'main' }}; then 'REMAINING_TESTS'; else 'RELEVANT_TESTS'` and then pass that value?

Reply:
`if` statements can't be used in the `ENV` part of actions, but there are some tertiary workarounds, one of which I've applied. It does need to be duplicated on both test jobs though...

Reply:
Thanks for looking in to this. While there is some duplication in this solution, i prefer the simplicity of it. It also makes it easier to use as an example for customers.